ci/terraform/shared/dynamodb.tf

```diff
@@ -196,7 +199,8 @@ resource "aws_dynamodb_table" "identity_credentials_table" {
   }
 
   server_side_encryption {
-    enabled = !var.use_localstack
+    enabled     = !var.use_localstack
```

Comment:
I know this isn't part of your change but as we don't use localstack anymore then these can just be set to true